src/settings.py

```diff
@@ -1,6 +1,7 @@
+import pygame
 import pygame.freetype
 import pytmx
-from src.import_checks import *
+from src.warnings import *
```

Comment:
Nope. You should not have renamed this file, as the name you chose shadows a built-in module.

Reply:
Oh, I didn't rename the file! I just had an error in my local project configuration, which is why the import was accidentally renamed. I'll change it immediately, I don't know why I didn't notice it before commiting